src/Microsoft.Azure.WebJobs.Extensions.Storage/AzureStorageDeploymentValidator.cs

```diff
@@ -1,51 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.WindowsAzure.Storage.Table;
-
-namespace Microsoft.Azure.WebJobs.Host
-{
-#if false // $$$ Anybody uses? This used to be called on Startup, call got removed in DI branch
-    /// <summary>
-    /// This helper addresses an issue in Azure Storage SDK where if certain assemblies are missing
-    /// the Storage SDK actually continues to work overall, but some features will silently fail.
-    /// This directly affects our StorageExceptionExtensions which rely on ExtendedErrorInfo being correctly
-    /// populated always. However, when certain assemblies are missing, ExtendedErrorInfo will be null.
-    /// See related issues:
-    /// <see href="https://github.com/Azure/azure-webjobs-sdk/issues/922"/>
-    /// <see href="https://github.com/Azure/azure-storage-net/issues/271"/> 
-    /// </summary>
-    internal static class AzureStorageDeploymentValidator
```

Comment:
No longer needed as described in https://github.com/Azure/azure-storage-net/issues/271